src/celluloid-application.c

```diff
@@ -307,6 +308,28 @@ options_handler(GApplication *gapp, GVariantDict *options, gpointer data)
 	return version?0:-1;
 }
 
+static gboolean
+local_cmdline_handler (	GApplication *gapp,
+			gchar ***arguments,
+			gint *exit_status)
+{
+	gint i = 1;
+	gchar **argv = *arguments;
+	CelluloidApplication *app = CELLULOID_APPLICATION(gapp);
+	g_clear_pointer(&app->mpv_options, g_free);
+	while(argv[i])
```

Comment:
It might be cleaner to locate `--` first (moving `argv` in the process), then process mpv options afterwards.